### ISSUE TITLE
-cifti-parcellate error when empty grayordinates exist (due to change in wb_command >= v1.4.0)

### ISSUE THREAD

**Issue by madisoth** [DCAN-Labs/dcan_bold_processing#14]
As of wb_command v1.4.0, the default behavior of -cifti-parcellate is to throw an error if data is missing for any grayordinate in a parcel. On a subject I was testing this caused the parcellation step to fail, although the pipeline (abcd-hcp-pipeline) was able to continue running to the end. 

There is the -legacy-mode flag to make -cifti-parcellate function as it did in wb_command < v1.4.0; I'm going to test if that fixes the issue:

```
If -legacy-mode is specified, parcels will be defined as the overlap between
a label and the data, with no errors for missing data vertices or voxels,
and empty parcels discarded.
```

https://www.humanconnectome.org/software/workbench-command/-cifti-parcellate

**Comment by madisoth** [comment]
resolved in PR #15, merge pending

**Comment by arueter1** [comment]
Since this will be resolved by the pull request I am closing this issue.
